mocha.js

```diff
@@ -943,6 +943,7 @@ Hook.prototype.error = function(err){
   this._error = err;
 };
 
+global.practical.mocha.Hook = Hook;
```

Comment:
Document - we put this in global scope so we can instantiate mocha classes for each run